.pipelines/singletenancy/aks/e2e-step-template.yaml

```diff
@@ -32,23 +32,23 @@ steps:
         echo "Upload CNI"
         echo "Deploying on Linux nodes"
         if [ "${{parameters.os}}" == "windows" ]; then
-          export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='linux' ARCH=${{ parameters.arch }} CNI_DROPGZ_VERSION=${{ parameters.version }})
-          echo "Dropgz image: $DROP_GZ_URL"
+          export CNI_IMAGE=$(make cni-image-name-and-tag OS='linux' ARCH=${{ parameters.arch }})
+          echo "CNI image: $CNI_IMAGE"
           envsubst < ./test/integration/manifests/cni/cni-installer-v1.yaml | kubectl apply -f -
           kubectl rollout status daemonset/azure-cni -n kube-system
           echo "Deploying on windows nodes"
-          export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='windows' ARCH=${{ parameters.arch }}  OS_VERSION=${{ parameters.os_version }} CNI_DROPGZ_VERSION=${{ parameters.version }})
-          echo "Dropgz image: $DROP_GZ_URL"
+          export CNI_IMAGE=$( make cni-image-name-and-tag OS='windows' ARCH=${{ parameters.arch }} OS_VERSION=${{ parameters.os_version }})
+          echo "CNI image: $CNI_IMAGE"
           envsubst < ./test/integration/manifests/cni/cni-installer-v1-windows.yaml | kubectl apply -f -
           kubectl rollout status daemonset/azure-cni-windows -n kube-system
         else
-          export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS=${{ parameters.os }} ARCH=${{ parameters.arch }} CNI_DROPGZ_VERSION=${{ parameters.version }})
-          echo "Dropgz image: $DROP_GZ_URL"
+          export CNI_IMAGE=$(make cni-image-name-and-tag OS=${{ parameters.os }} ARCH=${{ parameters.arch }})
+          echo "CNI image: $CNI_IMAGE"
```

Comment:
Is this missing `CNI_VERSION=$(make cni-version)` to generate the correct url? Or is it defaulting correctly as we are only using a singular cni image variety now?